netmaster/objApi/objapi_test.go

```diff
@@ -617,8 +631,11 @@ func TestTenantDelete(t *testing.T) {
 	checkCreateNetwork(t, false, "tenant1", "net1", "data", "vlan", "10.1.1.1/16", "10.1.1.254", 1, "", "")
 	checkCreateNetwork(t, false, "tenant1", "net2", "data", "vlan", "20.1.1.1/16", "20.1.1.254", 2, "", "")
 	checkCreateEpg(t, false, "tenant1", "net1", "group1", []string{}, []string{})
+	checkInspectEpg(t, false, "tenant1", "group1", 1)
```

Comment:
@gaurav-dalvi PR #478 has been merged, if you want to add integration test for this too..